R/waterseries.R

```diff
@@ -139,52 +147,85 @@ setMethod(f="waterseries", signature=c(object="data.frame"),
                                                   site=object$Site %>% unique,
                                                   park=object$Park %>% unique)
             
-           Xaxis<- object$Date
-           
+           Xaxis <- object$Date
+          
            OutPlot<-
               if(censored == TRUE){ 
                 if(deseason == FALSE){
-                  ggplot(object, aes(x = Xaxis, y = ValueCen, color = Censored, group = Censored))+
+                  ggplot(object, aes(x = Xaxis, y = adjValueCen, color = Censored, group = Censored))+
                     {if ("points" %in% layers) geom_point(size = sizes[1]) }+
-                    {if (is.numeric(assessment)) geom_hline(yintercept = assessment, color = assesscolor, linetype = "dashed", size = sizes[3])} +
+                    {if (is.numeric(assessment)) geom_hline(yintercept = assessment, color = assesscolor, 
+                                                            linetype = "dashed", size = sizes[3])} +
                     labs(title = title, y = yname, x = xname) +
-                    scale_color_manual(values = c("FALSE" = "black", "TRUE" = "red"),
-                    labels = c("True Value", "Censored"))+
+                    scale_color_manual(values = c("FALSE" = "blue", "TRUE" = "red"),
+                                       labels = c("FALSE" = "True Value", "TRUE"= "Censored"))+
                     theme_bw() +
                     theme(panel.grid = element_blank(), legend.title = element_blank(), legend.position = legend)
                   
                 } else if(deseason == TRUE){
-                  if(max(object$num_meas, na.rm = TRUE) < 5 | max(object$pct_true, na.rm = TRUE) < 0.5) 
-                    stop ("Must have at least 4 non-censored data points for at least one month for deseason plot.") 
                   
-                  
-                  df_mon <- object %>% filter(num_meas>=5 & pct_true>=0.5) %>% droplevels() %>% arrange(month)
-                  ggplot(df_mon, aes(x = Date, y = ValueCen, color = Censored, group = Censored))+
-                    {if ("points" %in% layers) geom_point(size = sizes[1]) }+
-                    {if (is.numeric(assessment)) geom_hline(yintercept = assessment, color = assesscolor, linetype = "dashed", size = sizes[3])} +
+                  df_mon <- object %>% filter(num_meas>=6) %>% droplevels() %>% arrange(month)
+                  print(head(df_mon))
```

Comment:
Is this debugging code?

Reply:
D'oh! Yes, I was printing to debug. You can delete the prints at line 168 and 204. Or I can. Just let me know.

Reply:
I just deleted the print lines and pushed the commit.